plugins/events-correlation-engine/src/test/java/org/opensearch/plugin/correlation/core/index/mapper/CorrelationVectorFieldMapperTests.java

```diff
@@ -191,7 +191,7 @@ public void testTypeParser_parse_fromCorrelationParamsContext_InvalidVectorSimil
 
     /**
      * test parseCreateField in CorrelationVectorFieldMapper
-     * @throws IOException
+     * @throws IOException ioexception
```

Comment:
Can we add more details for the IOException here? Also, I think the PR title is different than the changes in this PR?

Reply:
I am removing the extra protobuf-java file. The others came up as errors on my local precommit.

Reply:
@VachaShah Just FYI tests should be excluded from javadoc enforcement. I've seen cases where this doesn't work properly if building with JDK-14, so if that's the case for you I'd suggest moving to JDK 11 or 17.

Reply:
Ah thanks @andrross, that might be the case why it failed for me.